job.c

```diff
@@ -276,6 +277,40 @@ sig_dfl(void)
     signal(SIGPIPE, SIG_DFL);
 }
 
+char *
+make_mailbox_addr(char *displayname, char *mail_from, char *hostname)
+    /* Produce a "mailbox" header as per RFC5322 sec. 3.2.3
+     * <https://datatracker.ietf.org/doc/html/rfc5322#section-3.2.3>.
+     * Returns: the formatted mailbox header or NULL on errors like buffer
+     * overflow. */
+{
+    char *buf = NULL;
+    uint written = 0;
+    char need_anglebrackets = strlen(displayname);
```

Comment:
Yep, no need here for the assignment. The meaning is the same as below.

Reply:
Actually, what about using native `bool` type from `stdbool.h` here and elsewhere?

Reply:
good idea :+1:

Reply:
I just introduced a minimal set of changes for stdbool, mainly to avoid conflicts with `true` and `false` defined there. A whole review should make it in a different bug/PR.